packages/rest/src/lib/errors/DiscordAPIError.ts

```diff
@@ -0,0 +1,82 @@
+interface DiscordErrorFieldInformation {
+	code: string;
+	message: string;
+}
+
+interface DiscordErrorGroupWrapper {
+	_errors: DiscordError[];
+}
+
+type DiscordError = DiscordErrorGroupWrapper | DiscordErrorFieldInformation | { [k: string]: DiscordError } | string;
+
+export interface DiscordErrorData {
+	code: number;
+	message: string;
+	errors?: DiscordError;
+}
+
+function isErrorGroupWrapper(error: any): error is DiscordErrorGroupWrapper {
+	return Reflect.has(error, '_errors');
+}
+
+function isErrorResponse(error: any): error is DiscordErrorFieldInformation {
+	return typeof Reflect.get(error, 'message') === 'string';
```

Comment:
```suggestion
	return typeof error.message === 'string';
```

Reply:
Technically, error should be typed as unknown, thus needing to keep the Reflect.get code. Will fix 👍